schemas/org.gnome.shell.extensions.hijri-calendar.gschema.xml

```diff
@@ -35,6 +35,11 @@
       <summary>Startup Notification</summary>
       <description>Show notification on start up</description>
     </key>
+    <key name="arab-numerals" type="b">
+      <default>false</default>
```

Comment:
Since it was “true” in the past, can we have it to “true” by default? (refer to my other comment, it might be my because of my confusion!)